docs/FreeSWITCH-Explained/Troubleshooting-Debugging/Common-Errors.mdx

```diff
@@ -8,20 +8,17 @@ tags:
 
 # Common Errors 
 
- 
-
 ## Hangup Causes
 
 See [Hangup Causes](./Hangup-Cause-Code-Table.mdx#about) for hangup causes.
 
 ## SIP Error Codes
 
-Many error codes you may run into while trying to debug your freeswitch installation are actually SIP error codes. You can find more information about them at [RFC 3261](https://tools.ietf.org/html/rfc3261).
+Many error codes you may run into while trying to debug your FreeSWITCHⓇ installation are actually SIP error codes. You can find more information about them at [RFC 3261](https://tools.ietf.org/html/rfc3261).
```

Comment:
Not sure if using FreeSWITCHⓇ was actually OK'd. Did it here anyway. I can change it to FreeSWITCH™ if preferred.